packages/clerk-js/src/utils/syncDigest.ts

```diff
@@ -0,0 +1,26 @@
+import { SHA1 } from './sha1';
+
+type HashValueProps = {
+  sessionId: string | undefined | null;
+  userId: string | undefined | null;
+  orgId: string | undefined | null;
+};
+
+export const syncDigest = (values: HashValueProps) => {
```

Comment:
Can we make this more generic by having it accept a `string[]` or a `Record<string,string>`?
There's no reason to type the keys here I think